test/browser/render.js

```diff
@@ -243,209 +284,279 @@ describe('render()', () => {
 		expect(scratch.childNodes[0]).to.have.property('className', 'bar');
 	});
 
-	it('should apply style as String', () => {
-		render(<div style="top:5px; position:relative;" />, scratch);
-		expect(scratch.childNodes[0].style.cssText)
-			.that.matches(/top\s*:\s*5px\s*/)
-			.and.matches(/position\s*:\s*relative\s*/);
-	});
+	describe('style attribute', () => {
+		it('should apply style as String', () => {
+			render(<div style="top:5px; position:relative;" />, scratch);
+			expect(scratch.childNodes[0].style.cssText)
+				.that.matches(/top\s*:\s*5px\s*/)
+				.and.matches(/position\s*:\s*relative\s*/);
+		});
 
-	it('should only register on* functions as handlers', () => {
-		let click = () => {},
-			onclick = () => {};
+		it('should properly switch from string styles to object styles and back', () => {
```

Comment:
we'll want to revisit this if we decide to remove string style support (it's a preact-only feature)